org-mode/lib/Data/Org.hs

```diff
@@ -136,15 +137,17 @@ data Block
 -- This is content in the sub ~OrgDoc~.
 -- @
 data Section = Section
-  { sectionHeading :: NonEmpty Words
-  , sectionTags    :: [Text]
-  , sectionDoc     :: OrgDoc }
+  { sectionHeading  :: NonEmpty Words
+  , sectionTags     :: [Text]
+  , sectionClosed   :: Maybe Text  -- TODO Use a time type.
+  , sectionDeadline :: Maybe Text -- TODO Here too.
```

Comment:
Is it necessary to fully parse a time type here, or is `Text` sufficient?

Reply:
The manual indicates only day,
https://orgmode.org/manual/Inserting-deadline_002fschedule.html

By the way, this should be `sectionDeadline :: Maybe Day`, right?

Reply:
Scratch that, it *does* support time. In Emacs, when it prompts for day try entering `+2d 1pm`.

![image](https://user-images.githubusercontent.com/3998/116108036-8b1f2c00-a681-11eb-91e3-53a0cdda3457.png)

Source: https://stackoverflow.com/a/19538085/55246

Reply:
Deadlines can be assigned hours/minutes as well.

Reply:
I also misunderstood your question,

> Is it necessary to fully parse a time type here, or is Text sufficient?

Given Haskell's philosophy of making illegal states unrepresentable, I do think it would be good to represent this as a `LocalTime`, because we cannot have non-datetime values on this property anyway. That would be an error, which can be communicated to the library user somehow, eg:

```haskell
org :: Text -> Either OrgError OrgFileSource
```

Reply:
>  I'm just wondering if parsing that fully here is useful, or if we should leave it to the user.

Oh yea, I definitely would like to let the library parse as much of the org's (well-formed) plain-text formats as possible! :-) So I don't have to do any of the parsing myself.